tools/ci_build/github/azure-pipelines/mac-ios-ci-pipeline.yml

```diff
@@ -0,0 +1,10 @@
+jobs:
+- job: iOS_CI_on_Mac
+  pool:
+    vmImage: 'macOS-10.15'
+  timeoutInMinutes: 120
+  steps:
+    - script: |
+        sdkpath=`xcrun --sdk iphonesimulator --show-sdk-path`
+        python3 $(Build.SourcesDirectory)/tools/ci_build/build.py --build_dir iOS --ios --ios_sysroot $sdkpath --osx_arch x86_64 --apple_deploy_target 12.1 --use_xcode --config RelWithDebInfo --parallel
+      displayName: build iOS test cases and run them
```

Comment:
How about "Build onnxruntime for iOS x86_64 and run tests using simulator"?

Reply:
Who will own this pipeline and maintain it?

Reply:
How do we assign the owner for the existing pipelines? This pipeline should be followed in the same way since iOS now is the official feature of ORT.